applications/zpc/components/zwave_command_classes/src/zwave_command_class_meter_control.c

```diff
@@ -222,6 +230,11 @@ static sl_status_t zwave_command_class_meter_get(attribute_store_node_t node,
     get_frame->scale2 = (uint8_t)(scale - 8);
   }
 
+  if (supporting_node_version < 4) {
+    *frame_length = sizeof(ZW_METER_GET_V2_FRAME);
```

Comment:
shouldnt it be v3 ? or are their same

Reply:
They are the same:
https://raw.githubusercontent.com/SiliconLabs/UnifySDK/main/applications/zpc/components/zwave/zwave_definitions/include/ZW_classcmd.h
```
typedef struct _ZW_METER_GET_V2_FRAME_
{
    uint8_t   cmdClass;                     /* The command class */
    uint8_t   cmd;                          /* The command */
    uint8_t   properties1;                  /* masked byte */
} ZW_METER_GET_V2_FRAME;
```
```
typedef struct _ZW_METER_GET_V3_FRAME_
{
    uint8_t   cmdClass;                     /* The command class */
    uint8_t   cmd;                          /* The command */
    uint8_t   properties1;                  /* masked byte */
} ZW_METER_GET_V3_FRAME;
```

Reply:
you're right ...for now at least :)

that said for visibility I would use if  ( ...<=3 ) { sizeof( V3)  };